gradle.properties

```diff
@@ -2,5 +2,6 @@
 # Defaults to 'local' when the property doesn't exist, and gradle-local.properties should be ignored by version control
 # environmentName=
 
-geoServerHome=/var/tomcat/latest
-#geoServerHome=/space/geoserver/latest
\ No newline at end of file
+#geoServerHome=/var/tomcat/latest
+geoServerHome=/Users/rrudin/tools/geoserver-2.22.1
```

Comment:
This makes me wonder if the geoServerHome setting should be in gradle-local.properties.

Reply:
Eventually, yes, just hacking away for now